### PR TITLE
Found various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S *.po,./doc/sphinx/olddocs -L ans,bloc,elipse,hist,inout,maco,nd,shouldbe,te`

### CODE REVIEW

File: doc/sphinx/ui/mainviews/charview.rst [hunk 1]
```diff
@@ -499,10 +499,10 @@ including the start and end points of the line itself. If there are more than
 two intersections then the x, y, and total distance between the first and last
 intersections are shown before the point list.
 
-[0] indicates the starting point of the measure line, and the (x-y) co-ordinates
```
Comment (frank-trampe) [inline doc/sphinx/ui/mainviews/charview.rst:502]: This is a formatting choice, not a typographical error.

File: doc/sphinx/ui/mainviews/charview.rst [hunk 2]
```diff
@@ -499,10 +499,10 @@ including the start and end points of the line itself. If there are more than
...
+[0] indicates the starting point of the measure line, and the (x-y) coordinates
 of that point. Initially [1] is the end point; when it intersects with one
 spline then [1] becomes the first intersection point and [2] becomes the end
-point, and so on. The co-ordinates are followed by the x and y distances and
```
Comment (frank-trampe) [inline doc/sphinx/ui/mainviews/charview.rst:505]: This is a formatting choice, not a typographical error.

File: doc/sphinx/ui/mainviews/charview.rst (hunk 1 above)
Comment (luzpaz) [reply doc/sphinx/ui/mainviews/charview.rst]: Reverted @1f5831e

File: doc/sphinx/ui/mainviews/charview.rst (hunk 2 above)
Comment (luzpaz) [reply doc/sphinx/ui/mainviews/charview.rst]: Reverted in 1f5831e

File: fontforge/cvundoes.c [hunk 3]
```diff
@@ -3741,7 +3741,7 @@ return;
     /*  contain references, and the width of the original glyph is the same as*/
     /*  the width of the original referred character, then we should make sure */
     /*  that the new width of the glyph is the same as the current width of   */
-    /*  the referred char. We can't do this earlier because of foreward refs.  */
+    /*  the referred char. We can't do this earlier because of foreword refs.  */
```
Comment (frank-trampe) [inline fontforge/cvundoes.c:3744]: The original was lexically ambiguous, but it ought to be "forward".

File: fontforgeexe/contextchain.c [hunk 4]
```diff
@@ -1724,7 +1724,7 @@ static int WhichSections(struct contextchaindlg *ccd, GGadget *g) {
     } else if ( off==20 )
 	sections = 0x1;		/* Backtracking */
     else
-	sections = 0x4;		/* Foreward */
+	sections = 0x4;		/* Foreword */
```
Comment (frank-trampe) [inline fontforgeexe/contextchain.c:1727]: This ought to be "forward" instead.

**Comment by frank-trampe** [comment]
Everything else looks good.

File: fontforge/cvundoes.c (hunk 3 above)
Comment (luzpaz) [reply fontforge/cvundoes.c]: Fixed in  9f0a6b7

File: fontforgeexe/contextchain.c (hunk 4 above)
Comment (luzpaz) [reply fontforgeexe/contextchain.c]: Fixed in  9f0a6b7

**Comment by luzpaz** [comment]
Added requested revisions and rebased (for CI reasons)

**Pull request merged**